src/platform/plugins/shared/discover/public/application/types.ts

```diff
@@ -31,3 +32,8 @@ export interface SidebarToggleState {
   isCollapsed: boolean;
   toggle: undefined | ((isCollapsed: boolean) => void);
 }
+
+export interface DiscoverLatestFetchDetails {
+  abortController?: AbortController;
+  visContext?: UnifiedHistogramVisContext;
+}
```

Comment:
Tiny nit: personally I'd put this next to `DiscoverDataStateContainer` since it's never used outside of the main app. Not a strong opinion though.